src/dbt_score/cli.py

```diff
@@ -102,21 +118,32 @@ def lint(
         config.overload({"rule_namespaces": namespace})
     if disabled_rule:
         config.overload({"disabled_rules": disabled_rule})
+    if fail_project_under:
+        config.overload({"fail_project_under": fail_project_under})
+    if fail_any_model_under:
+        config.overload({"fail_any_model_under": fail_any_model_under})
 
     if run_dbt_parse:
         dbt_parse()
 
     try:
-        lint_dbt_project(
+        evaluation = lint_dbt_project(
             manifest_path=manifest, config=config, format=format, select=select
         )
+
     except FileNotFoundError:
         logger.error(
             "dbt's manifest.json could not be found. If you're in a dbt project, be "
             "sure to run 'dbt parse' first, or use the option '--run-dbt-parse'."
         )
         ctx.exit(2)
 
+    if (
+        any(x.value < config.fail_any_model_under for x in evaluation.scores.values())
```

Comment:
Not a big fan of using `x` personally, `score` would be more readable IMO

Reply:
That makes the line too long, and then spread over multiple lines I think we lose in readability